spec/lib/expansion_rules_spec.rb

```diff
@@ -82,19 +82,22 @@
     specify { expect(rules.expansion_fields(:placeholder_topical_event)).to eq(default_fields) }
     specify { expect(rules.expansion_fields(:role_appointment)).to eq(role_appointment_fields) }
     specify { expect(rules.expansion_fields(:service_manual_topic)).to eq(service_manual_topic_fields) }
-    specify { expect(rules.expansion_fields(:step_by_step_nav, :part_of_step_navs)).to eq(step_by_step_auth_bypass_fields) }
-    specify { expect(rules.expansion_fields(:step_by_step_nav, :related_to_step_navs)).to eq(step_by_step_auth_bypass_fields) }
-    specify { expect(rules.expansion_fields(:step_by_step_nav, :unspecified_link)).to eq(step_by_step_fields) }
-    specify { expect(rules.expansion_fields(:step_by_step_nav)).to eq(step_by_step_auth_bypass_fields) }
     specify { expect(rules.expansion_fields(:pages_secondary_to_step_nav)).to eq(default_fields) }
     specify { expect(rules.expansion_fields(:topical_event)).to eq(default_fields) }
 
+    specify { expect(rules.expansion_fields(:step_by_step_nav, link_type: :part_of_step_navs)).to eq(step_by_step_auth_bypass_fields) }
+    specify { expect(rules.expansion_fields(:step_by_step_nav, link_type: :part_of_step_navs, draft: false)).to eq(step_by_step_fields) }
+    specify { expect(rules.expansion_fields(:step_by_step_nav, link_type: :related_to_step_navs)).to eq(step_by_step_auth_bypass_fields) }
+    specify { expect(rules.expansion_fields(:step_by_step_nav, link_type: :related_to_step_navs, draft: false)).to eq(step_by_step_fields) }
+    specify { expect(rules.expansion_fields(:step_by_step_nav, link_type: :unspecified_link)).to eq(step_by_step_fields) }
+    specify { expect(rules.expansion_fields(:step_by_step_nav)).to eq(step_by_step_auth_bypass_fields) }
```

Comment:
Why has this been reordered - is it to make it alphabetical? Can that be done outside of this commit so that it's easier to see what's changed?